scripts/s3manage.py

```diff
@@ -474,28 +477,26 @@ def parse_arguments(argv):
             print('Please provide a full version path.')
             sys.exit(1)
 
-    bucket_name_env = 'S3_MF_GEOADMIN3_%s' % deploy_target.upper()
-    bucket_name = os.environ.get(bucket_name_env)
-    if bucket_name is None:
-        usage()
-        print('%s env variable is not defined' % bucket_name_env)
-        sys.exit(1)
+    bucket_name = 'mf-geoadmin3-%s-dublin' % deploy_target.lower()
     user = os.environ.get('USER')
-    profile_name = '{}_aws_admin'.format(user)
+    profile_name = None
+    if user is not None:
+        profile_name = '{}_aws_admin'.format(user)
```

Comment:
we're still using the profile_name? I thought we had instance roles now of mf1 dev and jenkins.
If we do, please remove the profile_name definition and adapt the connection.

Reply:
not on mf1-dev, the script must work in both cases.

Reply:
I'll add that in my JIRA ticket